tests/functional.mk

```diff
@@ -62,4 +84,25 @@ BGB_ADDITIONAL_GO_ENV := CGO_ENABLED=0
 
 include makelib/build_go_bin.mk
 
+# 1 - image
+# 2 - aci directory
+# 3 - ace validator
+define FTST_GENERATE_ACE_IMAGE
+$1: ACTOOL := $$(ACTOOL)
+$1: $2/manifest $2/rootfs/ace-validator | $2/rootfs
+	mkdir -p $2/rootfs/opt/acvalidator
+	# Set a consistent timestamp so we get a consistent hash
+	# TODO(jonboulle): make this cleaner..
+	for path in "$2/rootfs" $2/rootfs/ace-validator; do \
+		touch -a -m -d 1970-01-01T00:00:00Z "$$$${path}"; \
+	done
+	"$(ACTOOL)" build --overwrite "$2" "$1"
+
+CREATE_DIRS += $2 $2/rootfs
```

Comment:
Please replace `$2/rootfs` with `$$(call dir-chain,$2,rootfs/opt/acvalidator)`. This is to make sure that in future `make clean` will remove it.